Updated Monitoring and Logging info (#51761)
kubernetes/website: content/en/docs/tasks/debug/logging/_index.md

---
title: "Logging in Kubernetes"
description: Logging architecture and system logs.
weight: 20
---

> This doc contains a set of resources describing logging in kubernates. It will help in information like how to collect, access, and analyze logs in Kubernetes using built-in tools and popular logging stacks:

Reviewer: ```suggestion
This page provides resources that describe logging in Kubernetes. You can learn how to collect, access, and analyze logs using built-in tools and popular logging stacks:
```
